examples/allegro_hand/allegro_hand.cc

```diff
@@ -132,7 +131,7 @@ class AllegroHandExample : public TrajOptExample {
 
     // Add a model of the hand
     std::string sdf_file = idto::FindIDTOResourceOrThrow(
```

Comment:
nit, we should probably rename to `FindIdtoResourceOrThrow()` to match the Drake style guide (acronysms are also camel cased)

Reply:
sgtm, will do